src/utils/Cortex.js

```diff
@@ -46,6 +46,18 @@ export function cortexFetch(input, init) {
 
   return fetch(`${Config.cortexApi.path}${input}${queryFormat}`, requestInit)
     .then((res) => {
+      if (res.status === 401 || res.status === 403) {
```

Comment:
@petro97 I know this is a bit ugly but until we have our Cortex lib, we can catch all un-authenticated responses here and clear localStorage in order to fetch new tokens